src/Team-Services-Bot.AcceptanceTests/Features/Welcome.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: Welcome
+	In order to make the user feel welcome. We provide a welcome message.
+
+Background: 
+	Given A user 'Test User'
+	And A clean state
+
+@Acceptance
+Scenario: Starting a conversation for the first time with the bot.
+	Given I started a conversation
+	When I say 'Hi'
+	Then the bot should respond with 'Welcome Test User. I see this is the first team we speak.'
+
+@Acceptance
+Scenario: Starting a conversation for the second time with the bot.
+	Given I started a conversation
+	And The user has previously logged in into the account and team project 'config:TeamProjectOne'
```

Comment:
config:TeamProjectOne is a little trick to get data van the TestContext or the AppSettings. So we can configure on each environment what team project we can use.